pkg/apis/scheduling/v1alpha1/type.go

```diff
@@ -94,3 +94,8 @@ type QueueUnitList struct {
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []QueueUnit `json:"items"`
 }
+
+// QueueSuspend is a flag that instructs the job operator to suspend processing this job
+const QueueSuspend = "scheduling.x-k8s.io/suspend"
+// MultiClusterPlacement is the scheduling result of the multi cluster scheduler
+const MultiClusterPlacement = "schedule.multicluster.io/placement"
```

Comment:
schedule.multicluster.io/placement -> scheduling.multicluster.x-k8s.io/placement

Reply:
Queue itself is independent on underlying cluster environment. It could run on top of one cluster, or multi clusters. How about calling it as "const Placement = scheduling.x-k8s.io/placement" directly？ Or if it's actually not used in single cluster, then "MultiClusterPlacement = "schedule.multicluster.io/placement" is fine.

Reply:
I second @wsxiaozhang 's idea. On early discussions Alex and I had, users may run the queue scheduler outsides a cluster. It seems more generic to drop `MultiCluster`.

Reply:
@denkensk  Can you elaborate how this constant is going to be used? My understanding this const is used in multicluster scenarios only but I can be wrong.

Reply:
@Fei-Guo MultiClusterPlacement is only used in multi cluster scenarios now.  But I wonder if we need to make the name   more generic to adapt to the future single user cluster scenario.

Reply:
"future single user cluster scenario."  Yes, I am asking what could be the single user cluster case where the Placement annotation is needed (e.g., will Placement be node in single cluster case?)

Reply:
Specific scenarios may need to be discussed further. What I might think of is like placement maybe the name of node pool or az in the single cluster.

Reply:
Ok, sounds reasonable. I am fine with removing multicluster keyword but a better comment is preferred to explain the possible usage (can be done later. )